### PR TITLE
Added additional descriptions

### PR DESCRIPTION
**Pull request by chrisblakley**
This adds a few more descriptions and clarifies some others for WordPress solutions to Lighthouse detections.

### CODE REVIEW

**Comment by chrisblakley** [comment]
I signed it!

File: wordpress/audits.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     "id": "uses-responsive-images",
     "title": "Properly size images",
     "description": "Serve images that are appropriately-sized to save cellular data and improve load time. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/oversized-images).",
-    "secondaryDescription": "Upload images directly through the media library to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). "
+    "secondaryDescription": "Upload images directly through the media library to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage."
```
Comment (housseindjirdeh) [inline wordpress/audits.json]: Any chance there's a canonical link within WordPress that we can point to that describes full size/full width images?

File: wordpress/audits.json [hunk 2]
```diff
@@ -504,17 +505,21 @@
   {
     "id": "is-crawlable",
     "title": "Page isn’t blocked from indexing",
-    "description": "Search engines are unable to include your pages in search results if they don't have permission to crawl them. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/indexing)."
+    "description": "Search engines are unable to include your pages in search results if they don't have permission to crawl them. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/indexing).",
+    "secondaryDescription": "Make sure the Search Engine Visibility setting (under Reading) is not discouraging search engines from indexing this site by unchecking the box. Individual posts may be set to `noindex` by plugins."
```
Comment (housseindjirdeh) [inline wordpress/audits.json]:
```suggestion
    "secondaryDescription": "Make sure the Search Engine Visibility setting (under Reading) is not discouraging search engines from indexing this site by unchecking the box. Individual posts may be set to `noindex` by plugins. [Learn more](https://codex.wordpress.org/Settings_Reading_Screen)"
```

File: wordpress/audits.json (hunk 1 above)
Comment (chrisblakley) [reply wordpress/audits.json]: Sure is! We can link it to this: https://codex.wordpress.org/Inserting_Images_into_Posts_and_Pages#Image_Size

I'll update the pull request.

**Comment by chrisblakley** [comment]
Added link to WordPress codex for image sizes. I'm happy to write up a few more now that I have a feel for what you're looking for.

**Comment by housseindjirdeh** [comment]
Thanks again, these are all looking so great.

And yes please! It'll definitely be a continuous effort to get all the audit descriptions ready with WordPress, but things are off to a great start :)

**Pull request merged**